PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -174,6 +174,58 @@ def miniAOD_customizeCommon(process):
     for idmod in electron_ids:
         setupAllVIDIdsInModule(process,idmod,setupVIDElectronSelection)
 
+    # Adding puppi jets
+    process.load('CommonTools.PileupAlgos.Puppi_cff')
+    process.patCandidates += process.puppi
```

Comment:
I was making sure that puppi was run before the packcandidates were made. It should work without it.
slimmedJetsPuppi and slimmedMETsPuppi are saved due to 
        'keep *_slimmedJets*_*_*',
        'keep *_slimmedMETs*_*_*',
in slimming_cff